std/string.d

```diff
@@ -4717,8 +4717,16 @@ string[string] abbrev(string[] values) @safe pure
  * starting from 0.
  *
  * Tab characters are expanded into enough spaces to bring the _column number
- * to the next multiple of tabsize, and carriage returns and newlines reset the
- * running _column number back to 0.
+ * to the next multiple of tabsize.
+ * If there are multiple lines in the string, the column number of the last
+ * line is returned.
+ *
+ * Params:
+ *    str = string to be analyzed
+ *    tabsize = number of columns a tab character represents
```

Comment:
Should capitalize after `=`, otherwise the HTML output may look odd (the parameter name and description are separated into separate HTML table cells IIRC).